### ISSUE TITLE
Confirmation is not asked if the call is initiated from call history entry's menu

### ISSUE THREAD

**Issue by AnthillSudoku** [FossifyOrg/Phone#240]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Phone/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).

### Affected app version

1.2.0

### Affected Android/Custom ROM version

eOS 2.3 (android 13)

### Affected device model

samsung a52q

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Enable "show a call confirmation dialog before initiating a call" in the settings
2. Go to call history
3. Touch the three dots in the left of a call entry
4. Call

### Expected behavior

A confirmation dialog appears

### Actual behavior

Call start immediately

**Comment by antodc** [comment]
If I may add, the same thing happens (i.e. no confirmation is asked) when making calls from the contact list.

**Comment by naveensingh** [comment]
> making calls from the contact list.

Do we have a call option in the contact list?

**Comment by antodc** [comment]
I did as you suggested. Reinstalled from F-Droid. If I select a contact and then I tap the associated phone number, the call starts immediately without asking any confirmation.
I have Android 11.

**Comment by naveensingh** [comment]
You probably are have Fossify Contacts set as the default. Clicking a contact in Fossify Phone launches the default contact app, it doesn't initiate a call.

**Comment by antodc** [comment]
Sorry, I did not notice that when selecting a contact from the phone list another app is launched. I checked the option in the contact app and now it works as expected.
Maybe the two options in the two apps should be somehow coordinated. If the purpose of the option in the phone app is to avoid unvoluntary calls, I do not think for the final user really matter if another app is launched in the middle of the process.
